src/gtp_ip_pool.erl

```diff
@@ -63,14 +63,20 @@ init(Type, First, Last, Shift) when Last >= First ->
     lager:debug("init Pool ~p - ~p (~p)", [Start, End, Size]),
     init_table(FreeTid, Start, End),
 
-    State = #state{type = Type,
+    Id = inet:ntoa(int2ip(Type, First)),
+    State = #state{id = Id,
+		   type = Type,
 		   first = First,
 		   last = Last,
 		   shift = Shift,
 		   used = 0,
 		   free = Size,
 		   used_pool = UsedTid,
 		   free_pool = FreeTid},
+
+    exo_init_metrics(State),
+    exo_update_pool_size_gauge(State, Size),
```

Comment:
The size is in State, as free. So why have it as argument two?

Reply:
because sometimes free != size and it could lead to an errors in feature